igc.h

```diff
@@ -131,6 +131,43 @@ class IgcFormat : public Format
     rec_bad = 1,		// Bad record
   };
 
+  char* opt_enl{nullptr};
+  char* opt_tas{nullptr};
+  char* opt_vat{nullptr};
+  char* opt_oat{nullptr};
+  char* opt_trt{nullptr};
+  char* opt_gsp{nullptr};
+  char* opt_fxa{nullptr};
+  char* opt_siu{nullptr};
+  char* opt_acz{nullptr};
+  char* opt_gfo{nullptr};
+
+  QMap<IgcFormat::igc_ext_type_t, QString> ext_description_map = {
+    {IgcFormat::igc_ext_type_t::ext_rec_enl, QString("Engine Noise (ENL; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_tas, QString("True Airspeed (TAS; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_vat, QString("Total Energy Vario (VAT; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_oat, QString("Outside Air Temperature (OAT; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_trt, QString("True Track (TRT; default=0)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_gsp, QString("Ground Speed (GSP; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_fxa, QString("Fix Accuracy (FXA; default=1)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_gfo, QString("G Force? (GFO; default=0)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_siu, QString("# Of Sats (SIU; default=0)")},
+    {IgcFormat::igc_ext_type_t::ext_rec_acz, QString("Z Acceleration (ACZ; default=1)")},
+  };
```

Comment:
Can we just place the string literals directly in igc_args?

Reply:
I think I'm with you here. That sounds better.

There's generally too many maps and vectors hashes and lists for my liking right now. It's something I'd like to tackle in another PR with some kind of ext_data construct that would combine `ext_description_map`, `ext_option_map`, and `igc_extension_map`, if that works for you.